inst/templates/article.qmd

```diff
@@ -0,0 +1,15 @@
+---
+title: "{{{ vignette_title }}}"
+format: html
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library({{Package}})
+```
```

Comment:
As below. Don't include the knitr piece (I think?), and move "setup" to `#| label: setup`.